js/signaling.js

```diff
@@ -186,9 +201,15 @@
 			return;
 		}
 
-		$.post(OC.generateUrl('/apps/spreed/signalling'), {
-			messages: JSON.stringify(this.spreedArrayConnection)
+		$.ajax({
```

Comment:
Could you please avoid code duplication (here and `_sendMessageWithCallback`) and move all code that sends messages to the server into one method?